packages/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -105,9 +105,35 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
     valueToBigNumber
   )
 
+  /**
+   * Checks if the oldest report for a given token is expired
+   * @param token The token for which to check reports
+   */
+  async isOldestReportExpired(token: CeloToken): Promise<[boolean, Address]> {
+    const tokenAddress = await this.kit.registry.addressFor(token)
+    const response = await this.contract.methods.isOldestReportExpired(tokenAddress).call()
+    return response as [boolean, Address]
+  }
+
+  /**
+   * Removes expired reports, if any exist
+   * @param token The token to remove reports for
+   * @param numReports The number of expired reports to remove.
```

Comment:
can this be adapted to explain upper bound nature (ie numReports=5 can remove only 1 report and still be a successful transaction)

Reply:
Yes! Thanks for the clarity-suggestion :)